src/webview/js/js.js

```diff
@@ -636,17 +629,39 @@ const handleLongPress = (target: Element) => {
   // should; for example, multi-touch events.  Better would be to either find
   // a library we can use which strives to handle all that complexity, or
   // get long-press events from the platform.
-  if (!lastTouchEventTimestamp || Date.now() - lastTouchEventTimestamp < 500) {
-    return;
-  }
 
-  lastTouchEventTimestamp = 0;
+  /**
+   * This method is a little convoluted, since it aims to provide a solution
+   * to the long press event in-availability. If this solution is not
+   * provided, both the short presses and the long presses can trigger at the
+   * same time when a user tries to perform an action on a message.
+   *
+   * The first if checks if the person is clicking the link, and if
+   * the holding time is less than 500ms.
+   * If both of the conditions check, and the target of the click is a URL,
+   * this script returns a `sendMessage` with type `url`, else, script
+   * returns with `sendMessage` type `longPress`, and an href if the element
+   * the `longPress` is performed on is a link.
```

Comment:
This paragraph is just reciting the same information as the code. This kind of comment is almost always of negative value -- it doesn't really make the code any easier to understand (so there's little positive value to begin with); it adds more length and more things to read (which is a cost); and most dangerously, it is very likely to become untrue in the future when the code changes, and actively mislead someone.

Reply:
(The paragraph above it is much more helpful, though! That's information that isn't easy to get from just reading the code, and is higher-level so less sensitive to becoming wrong when the code changes.)

Reply:
I agree, I'll remove it.